src/main/java/scene_system/SceneManager.java

```diff
@@ -1,12 +1,25 @@
 package scene_system;
 
+import characters.CharacterInventoryFacade;
 import scene_system.Scene;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+
+/**
+ * Handles modifications and accessing of information from scene classes, and stores a hashmap
+ * of the scenes in the game, that can be referenced by their names in string form.
+ */
 public class SceneManager {
 
+    private HashMap<String, Scene> sceneList;
 
     public SceneManager(){}
```

Comment:
is this still needed?

Reply:
Yes in order for it to work we need to store all the scenes in scene manager

Reply:
so it didn't show the exact line i commented on, i meant the empty constructor